bika/lims/content/analysisrequest.py

```diff
@@ -2357,6 +2363,30 @@ def getVerifier(self):
                 verifier = actor
         return verifier
 
+    @security.public
+    def getVerifiersIDs(self):
+        """Returns the ids from users that have verified at least one analysis
+        from this Analysis Request
+        """
+        verifiers_ids = list()
+        for brain in self.getAnalyses():
+            verifiers = brain.getVerificators or ""
+            verifiers_ids += verifiers.split(",")
+        return list(set(verifiers_ids))
```

Comment:
Is the sort order important here?
If you want to keep it, then I would suggest the following code:

```python
    def uniquify_items(self, items):
        """Uniquify the items with sort order
        """
        unique = []
        for item in items:
            if item in unique:
                continue
            unique.append(item)
        return unique
```

Reply:
Sort order is not important here. I know there is that `getVerifer` function (that returns the "last" user to verify an analysis before the AR transitioned automatically to "verified"), but the only place where that function is used is in invoice and honestly, I feel is totally wrong. Thus, I tagged that function with `deprecated` and haven't gone further. Thinking loud... yes, maybe I could remove that function forever.... will open a new PR.